client/components/QueuedGraph.vue

```diff
@@ -1,93 +1,108 @@
 <template >
-  <line-chart 
-    :chart-data="datacollection"
-    :options="{
-      responsive: true, 
-      maintainAspectRatio: true,
-      title: {
-        display: false,
-        text: 'Queued'
-      },
-      legend: {
-        display: false,
-      },
-      animation: {
-        duration:0 // turn off annoying bouncing animation
-      },
-      scales: {
-        yAxes: [{
-          stacked: false, // QueuedGraph is not stacked
-          ticks: {
-            suggestedMax: 10
-          }
-        }]
-      }     
-    }"
-    >
-  </line-chart>
+  <div class="singleChart">
+    <h4 class="chart-title">Queued: {{this.total}}</h4>
+    <div id="queuedGraphLegend"></div>
+    <line-chart 
+      :chart-data="datacollection"
+      :options="{
+        responsive: true, 
+        maintainAspectRatio: true,
+        title: {
+          display: false,
+          text: 'Queued'
+        },
+        legend: {
+          display: false,
+        },
+        animation: {
+          duration:0 // turn off annoying bouncing animation
+        },
+        scales: {
+          yAxes: [{
+            stacked: false, // QueuedGraph is not stacked
+            ticks: {
+              suggestedMax: 10
+            }
+          }]
+        }     
+      }"
+      >
+    </line-chart>
+  </div>
 </template>
 
 <script>
   
  import LineChart from './LineChart.js';
  import { eventBus } from '../client';
  import { getBackgroundColorFor, getBorderColorFor, lineWidthInPixels, pointRadiusInPixels} from '../client'; 
- import { truncate} from '../pages/utilities.js'; 
+ import { truncate} from '../pages/utilities.js';
+ import { isPathIn} from '../pages/utilities.js';  
```

Comment:
Can be condensed see https://github.com/fnproject/ui/pull/37/files#diff-a3db016c87a936e401dcbc268b43db7cR40

Reply:
Done (on all four graphs)